pkg/k8s/pod_info_repo.go

```diff
@@ -15,7 +15,7 @@ import (
 
 const (
 	resourceTypePods        = "pods"
-	waitCacheSyncPollPeriod = 2 * time.Second
+	waitCacheSyncPollPeriod = 60 * time.Second
```

Comment:
Is this actually used as timeout for a request? Can you please provide the pointer?